src/plugins/graph/models/graph-model.ts

```diff
@@ -92,17 +92,17 @@ export const GraphModel = TileContentModel
     annotationSizesCache: new ObservableMap<string,RectSize>()
   }))
   .preProcessSnapshot((snapshot: any) => {
+    const newSnap = cloneDeep(snapshot);
+    // Remove connecting-lines adornment if found
+    if(snapshot?.adornments) {
+      newSnap.adornments = snapshot.adornments.filter((adorn: any) => adorn.type !== 'Connecting Lines');
```

Comment:
I wouldn't think you'd need the `any` here. Snapshots are generally typed by MST, so I would have thought you could just use:
```typescript
      newSnap.adornments = snapshot.adornments.filter(adorn => adorn.type !== 'Connecting Lines');
```
or even:
```suggestion
      newSnap.adornments = snapshot.adornments.filter(({ type }) => type !== 'Connecting Lines');
```

Reply:
I think I would need to define a type that represents the union of old and new properties on GraphModel in order to get away from `any` typing.